server/controllers/appointments/getAvailableAppointments/index.js

```diff
@@ -0,0 +1,53 @@
+const { getUnavailableTimes } = require('../../../database/queries');
+const { appointmentDateValidation, boomify } = require('../../../utils');
+
+const getAllProfileData = async (req, res, next) => {
+  try {
+    const { date } = req.params;
+
+    const isValid = await appointmentDateValidation.isValid({
+      appointmentDate: date,
+    });
+
+    if (!isValid) {
+      return next(boomify(400, 'Invalid Date', 'Please send a correct date'));
+      //   return next(boom.conflict('Invalided Date'));
+    }
+
+    const availableTime = [
+      '12:00:00',
+      '13:00:00',
+      '14:00:00',
+      '15:00:00',
+      '16:00:00',
+      '17:00:00',
+      '18:00:00',
+      '19:00:00',
+      '20:00:00',
+      '21:00:00',
+      '22:00:00',
+      '23:00:00',
```

Comment:
Edit these hours to be 08:00:00 to 18:00:00 and try to edit the appointment hours at the fakeData

Reply:
doen